### DISCUSSION TITLE
Laravel Dusk error: "Could not verify the hashed value's configuration."

### DISCUSSION THREAD

**Discussion by OptimumTasarim** [laravel/framework#48910] [Q&A] [answered]
I have a Laravel app which has Pest PHP installed and running without issue. I decided to use Laravel Dusk and installed it as instructed in the Dusk documentation.

- I use heavily Livewire for the app.
- Using different database for testing

After the installation, when I try to run my tests, got this weird error in the console, and couldn't find anything about it on google:

```
Could not verify the hashed value's configuration.
  at vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php:1329
            return Hash::make($value);
        }
        if (! Hash::verifyConfiguration($value)) {
            throw new RuntimeException("Could not verify the hashed value's configuration.");
        }
        return $value;
    }
```

How can I solve that problem? Helps appreciated.

**Comment by OptimumTasarim** [comment, marked as answer]
Ok, solved the issue... 

I have this config in my `config/hashing.php` file:
```
'bcrypt' => [
        'rounds' => env('BCRYPT_ROUNDS', 12),
    ],
```

and in `phpunit.xml` file have:

```
<env name="BCRYPT_ROUNDS" value="4"/>
```

So, changed the `value="4"` to `value="12"` as in the hashing config file and everything works like a charm again.

**Reply by kocoten1992** [reply]
thanks!

**Reply by padupe** [reply]
Helped me solve a problem on the CI/CD trail. Thank you very much!

**Reply by Andr1yk0** [reply]
Increasing bcrypt rounds will make your tests slower. You can try this approach https://prostocode.com/posts/how-to-fix-could-not-verify-the-hashed-values-configuration-error

**Reply by da-mask** [reply]
I wonder how long this would have taken me to figure out without this discussion, thanks!!

**Reply by wajeht** [reply]
I was also spinning wheels. thank you very much!

**Reply by aManNamedJed** [reply]
Thank you!

**Reply by bibrokhim** [reply]
Thank you!

**Reply by marensas999** [reply]
Check your `UserFactory` and shorten the hash for `password` if you are using this factory in your tests.

**Reply by chadlockett** [reply]
Much appreciated!

**Reply by ruslanashaari** [reply]
this helps! thanks

**Reply by benwrigley** [reply]
I had same thing after upgrading Homestead and reinstalling my project. Thank you!
